addon/components/suspense-component/index.hbs

```diff
@@ -0,0 +1,3 @@
+<div ...attributes>
```

Comment:
Is there a strong benefit to provide a wrapper element instead of just yielding and be completely transparent in the DOM?

Reply:
Not from the rendering engines perspective, not sure if @sunishsheth2009 had something else in mind though...

Reply:
Yes, I thought about this. The only reason I added this was to make it backwards compatible. Previously we didn't have `tagName:''`. This major version bump was just to drop support for ember versions. The API was going to be the same. If we feel that we can break this as well. I am happy to do that. What do you guys think?

Reply:
In general:

* it seems odd to _actually_ care about the existing wrapper div (e.g. I doubt many folks would notice the difference in practice). 
* it seems that we don’t _need_ the DOM element
* we are already prepping a breaking change release

To me, those things add up to removing the wrapper element...